Pipfile

```diff
@@ -6,6 +6,9 @@ requests = "*"
 [dev-packages]
 pytest = "*"
 pytest-cov = "*"
+behave = "*"
```

Comment:
why don't you put directly the version instead of * ?

Reply:
done